aws/src/integration/java/org/apache/iceberg/aws/s3/TestFlakyS3InputStream.java

```diff
@@ -129,16 +130,17 @@ public void testReadWithFlakyStreamExhaustedRetries(
   public void testReadWithFlakyStreamNonRetryableException(
       Map<String, String> aalProperties, IOException exception) {
     final S3FileIOProperties s3FileIOProperties = new S3FileIOProperties(aalProperties);
-    skipIfAnalyticsAcceleratorEnabled(
-        s3FileIOProperties, "Analytics Accelerator wraps IOException differently");
     assertThatThrownBy(
             () ->
                 testRead(
                     flakyStreamClient(new AtomicInteger(3), exception),
                     flakyStreamAsyncClient(new AtomicInteger(3), exception),
                     aalProperties))
         .isInstanceOf(exception.getClass())
-        .hasMessage(exception.getMessage());
+        .hasMessage(
```

Comment:
I think we should update our exception handling logic / update the test instead of having this hardcoded `AAL_ERROR_MESSAGE ` as if AAL changes this, the Iceberg tests start failing.